cmd/virt-handler/ipv6-nat.nft

```diff
@@ -0,0 +1,6 @@
+table ip6 nat {
+	chain prerouting	{ type nat hook prerouting priority -100; }
+	chain input 		{ type nat hook input priority 100; }
+	chain output  		{ type nat hook output priority -100; }
```

Comment:
Alignment is off here. In general, I'd use spaces instead of tabs for alignment purposes.

Reply:
I just copied the files like they were distributed by fedora. Can change it.

Reply:
> I just copied the files like they were distributed by fedora. Can change it.

They look fine in Fedora 34. Which makes me wonder, are the files identical in CentOS Stream 8? If not, perhaps we should grab the version from CentOS Stream instead of the one from Fedora.

Reply:
Do they exist on centos8 stream? I could not find them.

Since we used exactly these table definitions on all clusters, the content should be fine independent of the node OS.

Reply:
Let's leave them alone then :)